pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -6,13 +6,38 @@ import (
 	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/types/config"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var FeatureAppNS = []string{
+	"ingress-nginx",              // This is for Ingress, not feature app
+	"kube-system",                // Harvester, vSphere CPI, vSphere CSI
+	"cattle-system",              // AKS/GKE/EKS Operator, Webhook, System Upgrade Controller
+	"cattle-epinio-system",       // Epinio
+	"cattle-fleet-system",        // Fleet
+	"longhorn-system",            // Longhorn
+	"cattle-neuvector-system",    // Neuvector
+	"cattle-monitoring-system",   // Monitoring and Sub-charts
+	"rancher-alerting-drivers",   // Alert Driver
+	"cis-operator-system",        // CIS Benchmark
+	"cattle-csp-adapter-system",  // CSP Adapter
+	"cattle-externalip-system",   // External IP Webhook
+	"cattle-gatekeeper-system",   // Gatekeeper
+	"istio-system",               // Istio and Sub-charts
+	"cattle-istio-system",        // Kiali
+	"cattle-logging-system",      // Logging
+	"cattle-windows-gmsa-system", // Windows GMSA
+	"cattle-wins-system",         // Wins Upgrader
```

Comment:
Wins-upgrader doesn't have 2.7.0 chart either, it's for unsupported RKE1 Windows upgrades

Reply:
This is removed.